app/src/components/annotation/AnnotationInputLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { LabelProps } from "react-aria-components";
+import { css } from "@emotion/react";
+
+import { classNames } from "@arizeai/components";
+
+import { Label } from "@phoenix/components";
+import { EXPLANATION_LABEL_WIDTH } from "@phoenix/components/annotation/AnnotationInputExplanation";
```

Comment:
very minor nit - i tend to like centralizing constants so the import graph is a bit easier to understand.